.github/workflows/scheduled.yml

```
@@ -0,0 +1,52 @@
+permissions:
+  contents: read
+on:
+  push:
+    branches: [main]
+  pull_request:
+  schedule:
+    - cron:  '7 7 * * *'
```

I think is better to reduce the frequency to one day of the week, maybe on Friday or Monday.
```suggestion
    - cron:  '7 7 * * 1'
```

---

On some previous commits, checks failed, make those successful as well.